### PR TITLE
[oraclelinux] Changed tag from 8.5 to 8.6.

### PR DESCRIPTION
**Pull request by amsteinb-orcl**
Signed-off-by: Alan Steinberg <alan.steinberg@oracle.com>

### CODE REVIEW

**Comment by yosifkit** [comment]
Did you want to re-add the `8.5` section so that the tag can be rebuilt to point at the previous build? I think the built tests here might fail, but our builder will still have access to the old commit and do fine.

And then it can be removed once re-pushed to Docker Hub.

**Comment by amsteinb-orcl** [comment]
I unfortunately do not know the correct action here.  The latest container images are for the Oracle Linux 8.6 release. I did not realize until after the upload that the library file needed to be updated.

**Comment by yosifkit** [comment]
This PR will correctly update the `8.6` tag and we can just go ahead and merge, but the `8.5` tag will be stuck pointing at an image containing 8.6. Do you want the `8.5` tag corrected?

If we add an extra section like this temporarily, it should allow a rebuild and push to fix the `8.5` tag to be pointing at the correct older image. I took the commit ids from the values previous to https://github.com/docker-library/official-images/pull/12451.

```yaml

# temporary 8.5 tag fix
Tags: 8.5
Architectures: amd64, arm64v8
amd64-GitCommit: b2631b4c3944ae70c9463ae62aaa75a1a4877a16
arm64v8-GitCommit: ae688d9eef59795cd485bd9d9f43ce9015c3c461
```

**Comment by amsteinb-orcl** [comment]
I made the change as you suggested. Thank you.

**Comment by yosifkit** [comment]
It looks right to me and the failing builds are expected from my end. Those old commits are not referenced in the specified `GitFetch` branches, but they are still accessible via github directly (like https://github.com/oracle/container-images/commit/b2631b4c3944ae70c9463ae62aaa75a1a4877a16), so we should be safe to merge.

cc @tianon

File: library/oraclelinux [hunk 1]
```diff
@@ -9,10 +9,16 @@ amd64-GitCommit: 2190ef249f644a54672562adab58500744619a38
...
 
-Tags: 8.5, 8
+Tags: 8.6, 8
 Architectures: amd64, arm64v8
 Directory: 8
 
+# temporary 8.5 tag fix
+Tags: 8.5
+Architectures: amd64, arm64v8
+amd64-GitCommit: b2631b4c3944ae70c9463ae62aaa75a1a4877a16
```
Comment (tianon) [inline library/oraclelinux:19]: This one needs `Directory:` also :eyes:

```suggestion
Directory: 8
amd64-GitCommit: b2631b4c3944ae70c9463ae62aaa75a1a4877a16
```

**Pull request merged**